include/solvers/initial_conditions.h

```diff
@@ -108,6 +109,10 @@ namespace Parameters
     Functions::ParsedFunction<dim> cahn_hilliard =
       Functions::ParsedFunction<dim>(2);
 
+    // Path to the checkpointed average velocity profile
+    std::string checkpoint_folder;
+    std::string checkpoint_file_name;
```

Comment:
```suggestion
    // Path to the checkpointed average velocity profile
    std::string checkpoint_folder;
    std::string checkpoint_file_name;
```

I would maybe use more descriptive variable names like average_velocity_folder and average_velocity_file_name ?